.github/workflows/validate-cg-manifest.sh

```diff
@@ -37,6 +37,7 @@ ignore_no_source_tarball=" \
   initramfs \
   installkernel \
   javapackages-tools-meta \
+  kata-packages-uvm \
```

Comment:
minor question: do we need to list the subpackages here as well?

Reply:
Entries in here are per .spec and not per-package